tests/testsuite/script.rs

```diff
@@ -1370,6 +1370,47 @@ registry+https://github.com/rust-lang/crates.io-index#dep@1.0.0
         .run();
 }
 
+#[cargo_test(nightly, reason = "edition2024 hasn't hit stable yet")]
+fn script_as_dep() {
```

Comment:
Is there a reason this test wasn't added before the feature commit?

Reply:
Just want to make sure it already has an error for it. The feature is just to improve the error message.